wire-library/wire-runtime-swift/src/test/swift/sample/Dinosaur.swift

```diff
@@ -0,0 +1,121 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source: squareup.dinosaurs.Dinosaur in squareup/dinosaurs/dinosaur.proto
+import Foundation
+import Wire
+
+public struct Dinosaur {
+
+    /**
+     * Common name of this dinosaur, like "Stegosaurus".
+     */
+    public var name: String?
+    /**
+     * URLs with images of this dinosaur.
+     */
+    public var picture_urls: [String]
+    public var length_meters: Double?
+    public var mass_kilograms: Double?
+    public var period: Period?
+    public var unknownFields: Data = .init()
+
+    public init(
+        name: String? = nil,
+        picture_urls: [String] = [],
+        length_meters: Double? = nil,
+        mass_kilograms: Double? = nil,
+        period: Period? = nil
+    ) {
+        self.name = name
+        self.picture_urls = picture_urls
+        self.length_meters = length_meters
+        self.mass_kilograms = mass_kilograms
+        self.period = period
+    }
+
+}
+
+#if !WIRE_REMOVE_EQUATABLE
+extension Dinosaur : Equatable {
+}
+#endif
+
+#if !WIRE_REMOVE_HASHABLE
+extension Dinosaur : Hashable {
+}
+#endif
+
+#if swift(>=5.5)
+extension Dinosaur : Sendable {
+}
+#endif
+
+extension Dinosaur : ProtoMessage {
+    public static func protoMessageTypeURL() -> String {
+        return "type.googleapis.com/squareup.dinosaurs.Dinosaur"
+    }
+}
+
+extension Dinosaur : Proto2Codable {
+    public init(from reader: ProtoReader) throws {
```

Comment:
For the purposes of this sample code, should we `/* snip */` out the Proto/Foundation codable implementations?

Reply:
Yes, let's do that and then merge 🎸

Reply:
I went a different route and used some \<details\> tags for all three languages.